perllib/FixMyStreet/Cobrand/Merton/Waste.pm

```diff
@@ -2,12 +2,22 @@ package FixMyStreet::Cobrand::Merton::Waste;
 
 use Moo::Role;
 with 'FixMyStreet::Roles::Cobrand::SLWP';
+with 'FixMyStreet::Roles::Cobrand::Adelante';
 
 use FixMyStreet::App::Form::Waste::Report::Merton;
 use FixMyStreet::App::Form::Waste::Request::Merton;
 
 has lpi_value => ( is => 'ro', default => 'MERTON' );
 
+sub waste_check_staff_payment_permissions {
+    my $self = shift;
+    my $c = $self->{c};
+
+    return unless $c->stash->{is_staff};
+
+    $c->stash->{staff_payments_allowed} = 'cnp';
```

Comment:
What does 'cnp' stand for?

Reply:
Cardholder Not Present.

Reply:
It's a flag for Roles::Cobrand::Adelante to use to say if staff or not, so could really be anything - there's also "paye" which then doesn't redirect elsewhere but shows a form to fill in.